sdk/core/azure-core-test/src/test_proxy_manager.cpp

```diff
@@ -225,12 +224,50 @@ void TestProxyManager::SetProxySanitizer()
   Azure::Core::Url matcherRequest(m_proxy);
   matcherRequest.AppendPath("Admin");
   matcherRequest.AppendPath("SetMatcher");
-  const std::string matcherBody
-      = "{\"compareBodies\": false ,\"ignoreQueryOrdering\": true , \"ignoredHeaders\": "
-        "\"x-ms-copy-source,x-ms-proposed-lease-id,x-ms-lease-id,x-ms-file-change-"
-        "time,x-ms-file-creation-time,x-ms-file-last-write-time,x-ms-destination-"
-        "lease-id,x-ms-source-lease-id,x-ms-source-content-crc64,x-ms-content-crc64,x-ms-source-"
-        "content-md5,x-ms-content-md5,Content-MD5\"}";
+  std::string matcherBody;
```

Comment:
you're missing a lot of the headers, some of those are not only for storage but for attestation, keyvault etc.

Reply:
Oh, I thought the headers I removed were for storage.

Do you know which of those are necessary for attestation and keyvault?

Reply:
@gearama @LarryOsterman

Reply:
not of the top of my head, just run the pipelines with your changes, if they pass then you are correct and you can ignore my comment :)

Reply:
Attestation doesn't have any specific headers, beyond the standard ones like `x-ms-client-request-id`, etc.

I can't speak to keyvault.